packages/Form/Input/file/src/FileTable.js

```diff
@@ -67,24 +61,17 @@ const FileTable = ({
   return (
     <div className={componentClassName}>
       {Errors(errors)}
-      <table className="table table-striped table-bordered">
-        <tbody>
-          {values.map(
-            value =>
-              value ? (
-                <LineFile
-                  disabled={disabled}
-                  file={value.file}
-                  onClick={onClick}
-                  id={value.id}
-                  key={value.id}
-                />
-              ) : (
-                ''
-              )
-          )}
-        </tbody>
-      </table>
+      <ul className="af-form__file-list">
+        {values.map(value => (
```

Comment:
You can destructuring value as 
```suggestion
        {values.map(value => (({file, id})
```

Reply:
yes, you're right :)